retrybp/filters.go

```diff
@@ -110,6 +118,40 @@ func UnrecoverableErrorFilter(err error, next retry.RetryIfFunc) bool {
 	return next(err)
 }
 
+// WithDefaultRetryableCodes returns a list including the given error codes and
+// the default retryable error codes:
+//
+// 1. TOO_EARLY
+//
+// 2. SERVICE_UNAVAILABLE
+//
+// 3. TIMEOUT
+func WithDefaultRetryableCodes(codes ...int32) []int32 {
+	return append([]int32{
+		baseplate.ErrorCode_TOO_EARLY,
+		baseplate.ErrorCode_SERVICE_UNAVAILABLE,
+		baseplate.ErrorCode_TIMEOUT,
+	}, codes...)
+}
+
+// BaseplateErrorFilter returns true if the given error is a BaseplateError and
+// returns one of the given codes, other wise it calls the next filter in the
+// chain.
+func BaseplateErrorFilter(codes ...int32) Filter {
+	codeMap := map[int32]bool{}
```

Comment:
This works but it's kinda unorthodox . I'd prefer `codeMap := make(map[int32]bool)` or even better, `codeMap := make(map[int32]bool, len(codes))`.